int32[] id          # サーボID
int16[] angle       # 現在の角度 (サーボから受信した値)
int16[] time        # 移動時間 (サーボから受信した値)
int16[] speed       # 角速度 (サーボから受信した値)
int16[] load        # 負荷 (サーボから受信した値)
int16[] temperature # 温度 (サーボから受信した値)
uint16[] error      # エラー
